Tests/Invoke-PxGet.Tests.ps1

```diff
@@ -0,0 +1,239 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+BeforeAll {
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    $script:testRoot  = $null
+    $script:moduleList = @()
+    $script:failed = $false
+    $script:testNum = 0
+
+    function GivenPxGetFile
+    {
+        param(
+            [Parameter(Mandatory)]
+            [string] $Contents
+        )
+
+        New-Item -Path $testRoot  -ItemType 'File' -Name "pxget.json" -Value $Contents
+    }
+
+    function ThenError
+    {
+        param(
+            [Parameter(Mandatory)]
+            [string] $Matches
+        )
+    
+        $Global:Error[-1] | Should -Match $Matches
+    }
+
+    function ThenInstalled
+    {
+        param(
+            [Parameter(Mandatory)]
+            [string] $ModuleName,
+
+            [Parameter(Mandatory)]
+            [string] $Version,
+
+            [string] $InstallPath
+        )
+
+        if( $InstallPath )

```

Comment:
It looks like PowerShell 7 adds a `Prerelease` property to the object returned by `Test-ModuleManifest`, so you can test for the prerelease portion of the version. Refactor this test to be like this: 

```powershell
$prerelease = ''
if( $Version -match '^(.*)-(.*)$' )
{
    $Version = $Matches[1]
    $prerelease = $Matches[2]
}

if( -not $InstallPath )
{
    $InstallPath = $testRoot
}

$manifestPath = Join-Path -Path $InstallPath -ChildPath $ModuleName
$manifestPath = Join-Path -Path $manifestPath -ChildPath $Version
$manifestPath = Join-Path -Path $manifestPath -ChildPath $ModuleName
$manifestPath | Should -Exist
$manifest = Test-ModuleManifest -Path $manifestPath
# For PowerShell 5.1
if( -not ($manifest | Get-Member -Name 'Prerelease') )
{
    $manifest | Add-Member -Name 'Prerelease' -MemberType ScriptProperty -Value { $this.PrivateData['PSData']['Prerelease'] }
}

$manifest | Should -Not -BeNullOREmpty
$manifest.Name | Should -Be $ModuleName
$manifest.Version | Should -Be $Version
$manifest.Prerelease | Should -Be $prerelease
```